docs/developer/checkout.mdx

```diff
@@ -170,10 +184,311 @@ We get a newly created checkout object for which we return the ID, token, total
 }
 ```
 
-## Selecting a shipping method
+## Managing items
+
+### Adding a product to checkout
+
+To add an item to the cart, use [`checkoutLinesAdd`](developer/api-reference/mutations/checkout-lines-add.mdx). Total prices will be updated automatically:
```

Comment:
Would be good to include somewhere info that storefront should always fetch the prices from backend after any change in lines, shipping, or addresses as the values can be changed after applied texes.  It can be treat as something obvious but I would like to emphasize it.

Reply:
I'm not sure in which section it would fit now. Maybe at assigning address section?
I'll let know Zaiste that such info would be also in Saleor introduction tutorial 💪